### DISCUSSION TITLE
Support for Autonomous Flight to Target Location via MAVLink in iNav

### DISCUSSION THREAD

**Discussion by yangmoulalala** [iNavFlight/inav#9943] [Q&A]
Hello, thank you for your open source work. I know that iNav can use a MAVLink receiver, but I would like to ask if iNav, under conditions with positioning, can function like PX4's offboard mode, where it flies autonomously to a target location by receiving the target's position through MAVLink?

**Comment by stronnag** [comment]
INAV is an MSP based flight controller.
You can fly autonomously to a position received via MSP. This is documented on the wiki in a number of articles, for example  https://github.com/iNavFlight/inav/wiki/INAV-Remote-Management,-Control-and-Telemetry#follow-me-gcs-nav

**Reply by yangmoulalala** [reply]
Think you very much.it is useful for me.
